lib/gui/modules/analytics.js

```diff
@@ -53,11 +70,18 @@ analytics.config(($mixpanelProvider) => {
 
     /* eslint-enable camelcase */
 
-    electron: app.getVersion(),
+    electron: process.versions.electron,
     node: process.version,
     arch: process.arch,
-    version: packageJSON.version
+    version: packageJSON.version,
+    OSPlatform: os.platform(),
```

Comment:
Let's use lowercase, given that this is the beginning of the variable name, e.g: `osPlatform`.